### PR TITLE
fix: coinbase wallet login

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
coinbase wallet login is failing on prod app.

# How
Not sure about the changes, did console logging and found the response didn't need JSON.parse and was throwing error. Maybe it's needed in different version of coinbase wallet? Will confirm it with @vishnumad 

# Test Plan
- Test sign in with coinbase wallet is working

### CODE REVIEW

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

**Review by alantoa — APPROVED** [review]
LGTM✨

**Comment by vishnumad** [comment]
This is caused by a regression in Coinbase Wallet version 28.7.0. This will be resolved with version 28.8.0 that will roll out next week (September 12). I've added @intergalacticspacehighway to Testflight to verify that the 28.8 update resolves this. Might want to hold off on this fix or wrap the `JSON.parse` in a try catch and fallback to just returning the result, since this change will break once CB Wallet is updated.

File: packages/app/providers/wallet-mobile-sdk-provider.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export function WalletMobileSDKProvider({
       };
       const [res] = await makeRequest([action]);
       if (res.result) {
-        return JSON.parse(res.result);
+        return res.result;
```
Comment (vishnumad) [inline packages/app/providers/wallet-mobile-sdk-provider.tsx:90]: Maybe change to this?
```ts
try {
  return JSON.parse(res.result);
} catch (e) {
  return res.result;
}
```

**Comment by intergalacticspacehighway** [comment]
Confirmed, it's working with 28.8.8 coinbase wallet. Closing. We can do another release on 12!

**Pull request closed without merging**